### PR TITLE
Add Media queries to Perdido example to make it more responsive.

### PR DESCRIPTION
**Pull request by wldcordeiro**

### CODE REVIEW

File: perdido/section-styles.js [hunk 1]
```diff
@@ -74,11 +74,45 @@ window.styles = {
...
+    }
+  },
+
+  '@media (min-width: 800px)': {
+    fourUp: {
+      '& article': {
+        extend: perdido.column('1/3'),
+      }
+    },
+    reordered: {
```
Comment (wldcordeiro) [inline perdido/section-styles.js]: @kof  For some reason this nested rule is generating a different selector than the other "reordered" key and not applying the styles like the fourUp one does.

Comment (kof) [inline perdido/section-styles.js]: can you submit a pr with a failing test in jss repo?

**Comment by kof** [comment]
Just found this PR, whats the state here?

**Comment by kof** [comment]
merged

**Pull request merged**